### PR TITLE
Add shoeboxed.com

### PR DESCRIPTION
**Pull request by ELLIOTTCABLE**

### CODE REVIEW

File: _data/finance.yml [hunk 1]
```diff
@@ -47,6 +47,12 @@ websites:
       img: quickbooks.png
       tfa: No
 
+    - name: Shoeboxed
+      url: https://www.shoeboxed.com/
+      twitter: quickbooks
```
Comment (jamcat22) [inline _data/finance.yml]: Should be `twitter: shoeboxed`

**Comment by jamcat22** [comment]
The Twitter account for Shoeboxed is "shoeboxed" not "quickbooks".

**Comment by ELLIOTTCABLE** [comment]
@jamcat22 Fixed that. Feel free to squash these two commits, I wasn't near a computer to do so (GitHub's web-edit is so perfect for simple shit like this! :D )

**Comment by jamcat22** [comment]
Looks great! :+1:

**Pull request merged**